src/lib/constants.js

```diff
@@ -93,8 +93,7 @@ export const STATUS_SUCCESS = 'success';
 export const STATUS_VIEWABLE = 'viewable';
 
 // X-Rep-Hints for Representations API
-export const X_REP_HINT_ARCHIVE = '[json]';
-export const X_REP_HINT_BASE = '[3d][pdf][text][mp3]';
+export const X_REP_HINT_BASE = '[3d][pdf][text][mp3][json]';
```

Comment:
This rep will now be hinted for every file. Did we check with the conversion team regarding the performance impact of this change?

Reply:
> This rep will now be hinted for every file. Did we check with the conversion team regarding the performance impact of this change?

Yeah, they said it's ok.